src/ol_infrastructure/applications/ocw_site/Pulumi.applications.ocw_site.Production.yaml

```diff
@@ -0,0 +1,4 @@
+secretsprovider: awskms://alias/infrastructure-secrets-qa  # pragma: allowlist secret
```

Comment:
This stack got instantiated with the wrong secrets provider. You'll likely need to destroy and regenerate the stack. It should be `infrastructure-secrets-production` for the `.Production` stack.

Reply:
Oh, OK. I can recreate it. I was just reusing shell history to create the production environment.

Reply:
Hopefully fixed in d622b55

I recreated the stack.